test/hermes/string-locale.js

```diff
@@ -6,9 +6,9 @@
  */
 
 /*
-RUN: LC_ALL=en_US.UTF-8 _HERMES_TEST_LOCALE=en_US.UTF-8 %hermes -O -target=HBC %s \
+RUN: LC_ALL=en_US.UTF-8 _HERMES_TEST_LOCALE=en_US %hermes -O -target=HBC %s \
 RUN:            | %FileCheck --match-full-lines -check-prefix EN %s
-RUN: LC_ALL=tr_TR.UTF-8 _HERMES_TEST_LOCALE=tr_TR.UTF-8 %hermes -O -target=HBC %s \
+RUN: LC_ALL=tr_TR.UTF-8 _HERMES_TEST_LOCALE=tr_TR %hermes -O -target=HBC %s \
 RUN:            | %FileCheck --match-full-lines -check-prefix TR %s
 RUN: LC_ALL=UTF-8 _HERMES_TEST_LOCALE=UTF-8 %hermes -O -target=HBC %s \
```

Comment:
For consistency, since we know this won't do anything, perhaps we should remove the `_HERMES_TEST_LOCALE` from this one.

```suggestion
RUN: LC_ALL=UTF-8 %hermes -O -target=HBC %s \
```

Reply:
Done. Looking at it again, I've changed the tags under _HERMES_TEST_LOCALE from _ to -, as when the parser is introduced, it should error throw on underscores.